indico/core/db/sqlalchemy/protection.py

```diff
@@ -231,6 +231,13 @@ def remove_principal(self, principal, quiet=False):
                                            old_data=entry.current_data, quiet=quiet)
             self.acl_entries.remove(entry)
 
+    def get_inherited_acl(self, count_only=False):
+        own_acl = {entry.principal for entry in self.acl_entries}
+        parent_acl = self.protection_parent.get_access_list(skip_managers=True)
+        if count_only:
```

Comment:
Maybe always returning the list is not *that* bad... it's not huge anyway and the main concern with the initial version of the PR was the nested loop which is already gone